packages/react/src/Form/Text/TextInput.stories.tsx

```diff
@@ -0,0 +1,147 @@
+};
+
+export default meta;
+
+type Story = StoryObj<React.ComponentProps<typeof TextInput>>;
+
+export const TextInputStory: Story = {
+  name: "TextInput",
+  render: ({ onChange, ...args }) => (
+    <TextInput onChange={onChange} {...args} />
+  ),
+  args: {
+    classModifier: "required",
+    value: "John Doe",
+    placeholder: "Your name",
+    name: "name",
+    id: "nameid",
+    readOnly: false,
+    disabled: false,
+    autoFocus: false,
+    className: "",
+    type: "text",
+    label: "Your name",
+    message: "",
+    helpMessage: "Aide à la saisie",
+    forceDisplayMessage: false,
+    messageType: MessageTypes.error,
+    classNameContainerLabel: "col-md-2",
+    classNameContainerInput: "col-md-10",
+    "aria-disabled": false,
+  },
+  argTypes: {
+    type: {
+      options: TYPES,
+      control: { type: "select" },
+    },
+  },
+};
+
+export const TextInputErrorStory: Story = {
+  name: "TextInputError",
+  render: ({ onChange, ...args }) => (
+    <TextInput onChange={onChange} {...args} />
+  ),
+  args: {
+    classModifier: "required",
+    value: "John Doe",
+    placeholder: "Your name",
+    name: "name",
+    id: "nameid",
+    readOnly: false,
+    disabled: false,
+    autoFocus: false,
+    className: "",
+    type: "text",
+    label: "Your name",
+    message: "The field is required",
+    helpMessage: "Aide à la saisie",
+    forceDisplayMessage: true,
+    messageType: MessageTypes.error,
+    classNameContainerLabel: "col-md-2",
+    classNameContainerInput: "col-md-10",
+    "aria-disabled": false,
+  },
+  argTypes: {
+    type: {
+      options: TYPES,
+      control: { type: "select" },
+    },
+  },
+};
+
+export const TextInputSuccessStory: Story = {
+  name: "TextInputSuccess",
+  render: ({ onChange, ...args }) => (
+    <TextInput onChange={onChange} {...args} />
+  ),
+  args: {
+    classModifier: "required",
+    value: "John Doe",
+    placeholder: "Your name",
+    name: "name",
+    id: "nameid",
+    readOnly: false,
+    disabled: false,
+    autoFocus: false,
+    className: "",
+    type: "text",
+    label: "Your name",
+    message: "Success Message Type",
+    helpMessage: "Aide à la saisie",
+    forceDisplayMessage: true,
+    messageType: MessageTypes.success,
+    classNameContainerLabel: "col-md-2",
+    classNameContainerInput: "col-md-10",
+    "aria-disabled": false,
+  },
+  argTypes: {
+    type: {
+      options: TYPES,
+      control: { type: "select" },
+    },
+  },
+};
+
+export const TextInputRichLabelStory: Story = {
+  name: "TextInputRichLabel",
+  render: ({ onChange, ...args }) => (
+    <TextInput onChange={onChange} {...args} />
+  ),
+  args: {
+    classModifier: "required",
+    value: "John Doe",
+    placeholder: "Your name",
+    name: "name",
+    id: "nameid",
+    readOnly: false,
+    disabled: false,
+    autoFocus: false,
+    className: "",
+    type: "text",
+    label: (
+      <span>
+        Place name <i>optional</i>
```

Comment:
si tu veux mettre en évidence le optional, un em/strong est plus adapté. Et de manière général on interdit les balises i/b car elles n'ont aucun sens sémantique.

```suggestion
        Place name <em>optional</em>
```

Reply:
D'accord, j'avais juste fait un copié collé de https://github.com/AxaFrance/react-toolkit/blob/master/packages/Form/Input/text/README.md?plain=1#L246
Je vais corriger ça 👍